src/helpers.js

```diff
@@ -43,6 +43,13 @@ export function showError(givenMessage, givenDetail = null) {
   })
 }
 
+export function idsToIgnoredRules(ruleIds = []) {
+  return ruleIds.reduce((ids, id) => {
+    ids[id] = 0
```

Comment:
I think it's good to use `0` like you have, to support older versions of ESLint which didn't understand `off`, but maybe it would be better to make a const like `const RULE_OFF_SEVERITY = 0` to avoid the magic number?